test/auth_test.ts

```diff
@@ -170,6 +170,45 @@ describe('Auth', () => {
       it('systemAuthentication', () => testTokenAuthFlow(createFakePack({systemConnectionAuthentication: auth})));
     });
 
+    describe(`${AuthenticationType.MultiHeaderToken}`, () => {
+      const auth: Authentication = {
+        type: AuthenticationType.MultiHeaderToken,
+        headers: [
+          {
+            name: 'Header1',
+            description: 'description 1',
+          },
+          {
+            name: 'Header2',
+            description: 'description 2',
+          },
+        ],
+      };
+
+      const execTest = (pack: PackDefinition) => {
+        setupReadline(['token1', 'token2']);
+
+        doSetupAuth(pack);
+
+        sinon.assert.calledWithExactly(
+          mockPromptForInput,
+          'Enter the token to use for the "Header1" header for this Pack:\n',
+          {mask: true},
+        );
+        sinon.assert.calledWithExactly(
+          mockPromptForInput,
+          'Enter the token to use for the "Header2" header for this Pack:\n',
+          {mask: true},
+        );
+        sinon.assert.calledOnceWithExactly(mockPrint, 'Credentials updated!');
+
+        assertCredentialsFileExactly({headers: {Header1: 'token1', Header2: 'token2'}});
+      };
+
+      it('defaultAuthenticatio', () => execTest(createFakePack({defaultAuthentication: auth})));
```

Comment:
Nit: missing the `n` at the end of `defaultAuthenticatio`

Reply:
Thanks, fixed.